Libplanet/Net/NetMQTransport.cs

```diff
@@ -536,6 +536,12 @@ public void ReplyMessage(Message message)
             _replyQueue.Enqueue(message);
         }
 
+        public async Task CheckAllPeersAsync(CancellationToken cancellationToken)
+        {
+            KademliaProtocol kp = (KademliaProtocol)Protocol;
```

Comment:
(Trivial) I think `var` is enough because we cast `Protocol` as `KademliaProtocol`.

```suggestion
            var kp = (KademliaProtocol)Protocol;
```